addons/rose/addon/styles/rose/components/toolbar/_info.scss

```diff
@@ -0,0 +1,17 @@
+@use '../../variables/sizing';
+@use '../../variables/media';
+@use '../../utilities/type';
+
+.rose-toolbar {
+  .rose-toolbar-info {
+    @include type.type(button);
+    color: var(--ui-gray);
+    position: relative;
+    display: inline-block;
+    vertical-align: middle;
+    white-space: nowrap;
```

Comment:
Is `nowrap` needed? Will this cause text to break out of its container? 

How about clipping text that is too long with `text-overflow: ellipsis`?